nautobot/core/events/__init__.py

```diff
@@ -12,6 +16,43 @@
 logger = logging.getLogger(__name__)
 
 
+def load_event_brokers(nautobt_event_brokers):
+    """Process plugins and log errors if they can't be loaded."""
+    for publisher_name, publisher in nautobt_event_brokers.items():
+        options = publisher.get("OPTIONS", {})
+        topics = publisher.get("TOPICS", {})
+        if not isinstance(topics, dict):
+            raise EventPublisherImproperlyConfigured(
+                f"Malformed Event Publisher Settings: Expected `TOPICS` to be a 'dict', instead a '{type(topics).__name__}' was provided"
+            )
+        include_topics = topics.get("INCLUDE", ["*"])
```

Comment:
Probably makes sense to default these to `None` here since the EventBroker class itself knows how to handle `None` as inputs, just to avoid duplication of defaults definitions? I know it'd make the type checking below a bit more complex but I think it's worthwhile.